src/crypto.ts

```diff
@@ -1,7 +1,11 @@
 import blake from "blakejs";
-import { RegistryValue } from "./registry";
-import { FileID } from "./skydb";
+import { RegistryEntry } from "./registry";
 import { stringToUint8Array } from "./utils";
+import { pki } from "node-forge";
+
+export type PublicKey = pki.ed25519.NativeBuffer;
```

Comment:
does this make sense to export? if so, shouldn't we re-export these in index.ts?